cmd/php/webconfig/main.go

```diff
@@ -211,6 +229,14 @@ func fpmConfig(layer string, addNoDecorateWorkers bool, overrides webconfig.Over
 		fpm.ListenAddress = defaultFlexAddress
 	}
 
+	if overrides.PHPFPMDynamicWorkers {
```

Comment:
Thanks for the hint on this, I'll take a look properly soon & ensure it functions the same.

I'll also double check the other stuff recently added, like the static files & front controller options are also replicated here, if applicable.

Reply:
This should be sorted now, I've replicated all of the extra config options into `supervisor`.

I've also moved the environment variable stuff into `webconfig`, so it can be called from both places, rather than just copy/pasting the checks into both places.

Any better @sonnysasaka ?